packages/react/src/components/Pagination/Pagination.tsx

```diff
@@ -20,6 +20,7 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
   onFirstPageClick?: () => void;
   onLastPageClick?: () => void;
   tooltipPlacement?: Placement;
+  thin?: boolean;
```

Comment:
For my own edification, why did you decide to go with a modifier prop, instead of `variant="thin"`?

Reply:
Imagine you could have both a variant AND thin, so you would want to be able to combine both so `thin` acts better as a modifier. This is how we've implemented it for Button, MenuBar, Tabs, etc. There's no variant for pagination right now, but this keeps it in sync with other components and allows for flexibility in the future if we do ever need it.